### PR TITLE
Correctly show vobject title in trash bin

### LINKED ISSUE #3330: Deleted tasks show up as `Untitled event`

### PR DESCRIPTION
**Pull request by raimund-schluessler**
Screenshot 2021-07-24 at 20-59-28 Week 30 of 2021 - Calendar - Nextcloud

Closes #3330.

### CODE REVIEW

**Review by tcitworld — APPROVED** [review]

File: src/components/AppNavigation/CalendarList/Trashbin.vue [hunk 1]
```diff
@@ -131,10 +131,9 @@ export default {
 				color: calendar.color ?? uidToHexColor(calendar.displayname),
 			}))
 			const formattedCalendarObjects = this.objects.map(vobject => {
-				let eventSummary = t('calendar', 'Untitled event')
+				let eventSummary = t('calendar', 'Untitled item')
```
Comment (raimund-schluessler) [inline src/components/AppNavigation/CalendarList/Trashbin.vue:134]:
```suggestion
				let eventSummary = t('calendar', 'Untitled element')
```

Comment (raimund-schluessler) [reply src/components/AppNavigation/CalendarList/Trashbin.vue:134]: Just to be in line with https://github.com/nextcloud/calendar/pull/3365#discussion_r676362126

**Review by ChristophWurst — APPROVED** [review]
:pray:

**Pull request merged**